pkg/cmd/cli/cmd/set/helper.go

```diff
@@ -42,6 +42,12 @@ func handlePodUpdateError(out io.Writer, err error, resource string) {
 			if all && match {
 				return
 			}
+		} else {
+			fmt.Fprintf(out, "error: %s %q is invalid\n\n", errorDetails.Kind, errorDetails.Name)
```

Comment:
We should be using the multi-error output from kcmdutil here (or putting a method in place there that does this consistently).  I.e. we want a "print err with causes" helper in kcmdutil.